Automattic/wp-calypso
# Editing toolkit: Update Newspack to 1.8.0

#### Changes proposed in this Pull Request

* Updates newspack-blocks to 1.8.0

This version includes the bug fix for Automattic/newspack-blocks#510. But we need to upgrade the version used on wpcom for our customers to get the fix too (tracked here #43049)

I've done the minimum version bump required to get the bug fix to play it safe, since I'm not very familiar with the recent changes in the blocks and how disruptive they'll be. But I could be convinced that this is the wrong approach.

Release notes for all the 1.7 to 1.8 changes:

- https://github.com/Automattic/newspack-blocks/releases/tag/v1.7.1
- https://github.com/Automattic/newspack-blocks/releases/tag/v1.7.2
- https://github.com/Automattic/newspack-blocks/releases/tag/v1.8.0

#### Testing instructions



* Load posts without newspack blocks and make sure there are no scripts loaded in the front-end
* On posts that contain newspack blocks, make sure they still work as expected

Fixes #43049
